components/survey/List/surveysPerPage.js

```diff
@@ -0,0 +1 @@
+export const surveysPerPage = 5;
```

Comment:
@carryall @tyrro open question: Should we name this file `constant.js`? 🤔 (I'm fine with both, just curious to know your ideas :) )

Reply:
that sounds make sense to me 🤔

Reply:
IMHO, creating a separate `constants` folder makes more sense than creating a file `constant.js` inside a nested folder structure as we have here (`components/survey/List/surveysPerPage.js`). But in this case, I'd keep it as it is as this constant belongs to this particular component only, What do you say?

Reply:
as usual, it depends on the context (if you have many or few constants :) ).
In this case, it definitely makes sense. We could even call the file something like "constants/appConfig.js"

Reply:
sure, update the file to `constants/surveyConfig.js` in 94fcea02435d99567a6cbc80d3247a52539f0369